evap/evaluation/tests/tools.py

```diff
@@ -273,3 +257,75 @@ def assert_no_database_modifications(*args, **kwargs):
             lower_sql = query["sql"].lower()
             if not any(lower_sql.startswith(prefix) for prefix in allowed_prefixes):
                 raise AssertionError("Unexpected modifying query found: " + query["sql"])
+
+
+@tag("live-server")
+class LiveServerTest(SeleniumTestCase):
+    external_host = os.environ.get("TEST_HOST", "") or None
+    browser = "firefox"
+    selenium_hub = None
+    headless = True
+    window_size = (1920, 3080)
+
+    serialized_rollback = True
+
+    def _create_test_user(self):
+        """Create a default test user."""
+        test_user = baker.make(
+            UserProfile, email="evap@institution.example.com", groups=[Group.objects.get(name="Manager")]
+        )
+        test_user_password = "evap"  # nosec
+        test_user.set_password(test_user_password)
+        test_user.save()
```

Comment:
```suggestion
        """Create a default test user."""
        test_user = baker.prepare(
            UserProfile, email="evap@institution.example.com", groups=[Group.objects.get(name="Manager")]
        )
        test_user.set_password("evap")  # nosec
        test_user.save()
```
doesn't hit the database twice